src/bootstrap/initializer.js

```diff
@@ -1,30 +1,46 @@
 import React, { PureComponent } from 'react'
-
-import { web3 } from './kleros'
-import RequiresMetaMask from './requires-meta-mask'
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
+import * as walletActions from '../actions/wallet'
+import * as walletSelectors from '../reducers/wallet'
+import { eth } from './kleros'
+import { renderIf } from '../utils/react-redux'
+import RequiresMetaMask from '../components/requires-meta-mask'
```

Comment:
I like components being PascalCase

Reply:
Do you mean I should change the filename/path? Or is this good?

The only reason I made the filename kebab-case was so we can enforce a consistent filename pattern with the linter. Otherwise it would be up to us to make sure only component files have PascalCase filenames. I guess it's not that bad. What do you think?